com.unity.ml-agents.extensions/Documentation~/Match3.md

```diff
@@ -0,0 +1,60 @@
+# Match-3 Game Support
+
+We provide some utilities to make integration with Match-3 games.
+
+## AbstractBoard class
+The AbstractBoard is the bridge between ML-Agents and your game. It allows ML-Agents to
+* ask your game what the "color" at a row and column is
+* ask your game whether a move is allowed
+* request that your game make a move
+These are handled by implementing the `GetCellType()`, `IsMoveValid()`, and `MakeMove()` abstract methods.
+
+The AbstractBoard also tracks the number of rows, columns, and potential piece types that the board can have.
+
+### `public abstract int GetCellType(int row, int col)`
+Returns the "type" of piece at the given row and column.
```

Comment:
I think you mentioned color before. Maybe say "type"/"color" instead of just "type"